keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/EventControllerTests.kt

```diff
@@ -51,9 +51,10 @@ import java.time.Instant
 @SpringBootTest(
   classes = [KeelApplication::class, MockEurekaConfiguration::class, MockTimeConfiguration::class],
   properties = [
-    "clouddriver.baseUrl=https://localhost:8081",
-    "orca.baseUrl=https://localhost:8082",
-    "front50.baseUrl=https://localhost:8083"
+    "clouddriver.base-url=https://localhost:8081",
```

Comment:
we might be able to add a src/test/resources/application.properties with all these values in it and not have to add them to all the spring based tests

Reply:
Figured it out, thanks for the suggestion! That is so much better.